symbols.js

```diff
@@ -64,6 +69,11 @@ const symbols = [
         name: "Plus-minus",
         searchTerms: ["plus", "minus"],
     },
+    {
+        glyph: "−",
+        name: "minus",
+        searchTerms: ["minus"],
+    },
```

Comment:
Are there seriously keyboards without a minus symbol?  What about plus?

Reply:
The keyboard - (dash or hyphen) is different typologically from the − minus.